control/content/index.html

```diff
@@ -3,263 +3,124 @@
 <head lang="en">
     <meta charset="UTF-8">
 
-	<!-- CSS -->
+    <!-- CSS -->
     <link href="../../../../styles/helper.css" rel="stylesheet">
+    <link href="../../resources/peopleplugin.css" rel="stylesheet">
 
-	<!-- JS -->
+    <!-- JS -->
+    <script src="../../../../scripts/jquery/jquery-1.11.2.min.js"></script>
+    <script src="../../resources/bootstrap.min.js"></script>
```

Comment:
understood however we already provide it to you here https://github.com/BuildFire/sdk/blob/master/scripts/angular/ui-bootstrap.min.js